core/src/main/resources/lib/hudson/artifactList_de.properties

```diff
@@ -21,3 +21,6 @@
 # THE SOFTWARE.
 
 View=anzeigen
+view=anzeigen
```

Comment:
you mean `view` and `View` are used to translate two different things? that was not clear from your comment